.travis.yml

```diff
@@ -39,13 +39,16 @@ before_install:
   # PrestaShop configuration
   - cp tests/parameters.yml.travis app/config/parameters.yml
 
+
 notifications:
   hipchat: ec4e21c5eb82066ba8be5fd1afefde@1184657
 
 script:
   - composer install --prefer-dist --no-interaction --no-progress
   - bash tests/check_file_syntax.sh
   - bash travis-scripts/install-prestashop
+  - php app/console lint:twig src
```

Comment:
Twig is dependant form database now, since we loop into every "enabled" module :)